translations/ui-data-import/en.json

```diff
@@ -20,13 +20,15 @@
   "today": "today",
   "loading": "Loading",
   "undo": "Undo",
+  "delete": "Delete",
 
   "uploadTitle": "Drag and drop to start a new data import job",
   "activeUploadTitle": "Drop to start import",
   "uploadBtnText": "or choose files",
   "uploadingPaneTitle": "Files",
   "uploadingMessage": "Uploading",
-  
+  "uploadFileError": "Error: Upload file",
```

Comment:
Error uploading file

Reply:
Going to raise the question to Ann-Marie about this. The current message is retrieved from mockup and for the moment is the source of truth.